roles/deluge/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+deluge_enabled: false
+deluge_available_externally: "false"
+
+# directories
+deluge_config_directory: "{{ docker_home }}/deluge/config"
+deluge_download_directory: "{{ downloads_root }}"
+deluge_watch_directory: "{{ torrents_root }}"
+
+# uid / gid
+deluge_user_id: "0"
+deluge_group_id: "0"
+
+# error logging
+deluge_log_level: "error"
+
+# network
+deluge_port: "32030"
```

Comment:
I'd prefer this is set to the default port for now, and we address port allocations in the issue discussing it